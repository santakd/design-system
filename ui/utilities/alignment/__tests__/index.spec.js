/* eslint-env jest */
import React from 'react';
import createHelpers from '../../../../jest.helpers';
const { matchesMarkup } = createHelpers(__dirname);

describe('renders absolute center utility', () => {
  it('renders utility', () =>
    matchesMarkup(
      <div className="slds-align_absolute-center" style={{ height: '5rem' }}>
        This content will be positioned in the absolute center of its container
      </div>
    ));

  it('renders utility nested in a flex container', () =>
    matchesMarkup(
      <div className="slds-grid" style={{ height: '5rem' }}>
        <div className="slds-align_absolute-center">
          This content will be positioned in the absolute center of its
          container
        </div>
      </div>
    ));
});
