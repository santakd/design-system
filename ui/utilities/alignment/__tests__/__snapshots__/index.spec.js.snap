// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders absolute center utility renders utility 1`] = `
<div
  className="slds-align_absolute-center"
  style={
    Object {
      "height": "5rem",
    }
  }
>
  This content will be positioned in the absolute center of its container
</div>
`;

exports[`renders absolute center utility renders utility nested in a flex container 1`] = `
<div
  className="slds-grid"
  style={
    Object {
      "height": "5rem",
    }
  }
>
  <div
    className="slds-align_absolute-center"
  >
    This content will be positioned in the absolute center of its container
  </div>
</div>
`;
